CBLClient/Apps/CBLTestServer-Java-Desktop/build.gradle

```diff
@@ -53,6 +56,7 @@ java {
 }
 
 dependencies {
+    compile 'commons-daemon:commons-daemon:1.2.2'
```

Comment:
"compile" is deprecated.  I think you should be using "implementation".  I know it is easy to confuse with "compileOnly" but I'm pretty sure you want the latter.